plugnpy/metric.py

```diff
@@ -29,15 +14,49 @@ class Metric(object):
         - warning_threshold -- Warning threshold for the Metric (default: '')
         - critical_threshold -- Critical threshold for the Metric (default: '')
         - display_format -- Formatting string to print the Metric (default: "{name} is {value} {unit}")
-        - convert_metric -- Whether to convert the metric in to a friendly form (default: True)
         - display_name -- Name to be used in friendly output (default: value of name)
+        - display_in_summary -- Whether to print the metric in the summary (default: True)
         - display_in_perf -- Whether to print the metric in performance variable (default: True)
+        - convert_metric -- Whether to convert the metric in to a friendly form (default: False)
```

Comment:
this comment can be improved - doesn't really describe what `convert_metric` is doing